airbyte-integrations/connectors/source-spree-commerce/manifest.yaml

```diff
@@ -0,0 +1,1094 @@
+    authenticator:
+      type: OAuthAuthenticator
+      client_id: "{{ config[\"client_id\"] }}"
+      grant_type: client_credentials
+      client_secret: "{{ config[\"client_secret\"] }}"
+      refresh_request_body: {}
+      token_refresh_endpoint: /spree_oauth/token
+
+streams:
+  - $ref: "#/definitions/streams/List Users"
+  - $ref: "#/definitions/streams/Variants"
+  - $ref: "#/definitions/streams/List Products"
+  - $ref: "#/definitions/streams/Payments"
+  - $ref: "#/definitions/streams/Payment Methods"
+  - $ref: "#/definitions/streams/List Orders"
+  - $ref: "#/definitions/streams/Menus"
+  - $ref: "#/definitions/streams/Addresses"
+  - $ref: "#/definitions/streams/Classifications"
+  - $ref: "#/definitions/streams/Adjustments"
+  - $ref: "#/definitions/streams/CMS Pages"
+  - $ref: "#/definitions/streams/CMS Sections"
+  - $ref: "#/definitions/streams/Shipments"
+  - $ref: "#/definitions/streams/Shipping Categories"
+  - $ref: "#/definitions/streams/States"
+  - $ref: "#/definitions/streams/Stock Items"
+  - $ref: "#/definitions/streams/Stock Locations"
+  - $ref: "#/definitions/streams/Stock Credit Categories"
+  - $ref: "#/definitions/streams/Store Credit Types"
+  - $ref: "#/definitions/streams/Store Credits"
+  - $ref: "#/definitions/streams/Tax Categories"
+  - $ref: "#/definitions/streams/Tax Rates"
+  - $ref: "#/definitions/streams/Wished Items"
+  - $ref: "#/definitions/streams/Wishlists"
+  - $ref: "#/definitions/streams/Zones"
+  - $ref: "#/definitions/streams/Webhook Subscribers"
+  - $ref: "#/definitions/streams/Taxonomies"
+  - $ref: "#/definitions/streams/Taxons"
+
+spec:
+  type: Spec
+  connection_specification:
+    type: object
+    $schema: http://json-schema.org/draft-07/schema#
+    required:
+      - base_url
+      - client_id
+      - client_secret
+    properties:
+      base_url:
+        type: string
+        order: 0
+        title: Base_URL
+      client_id:
+        type: string
+        order: 1
+        title: Client ID
+        airbyte_secret: true
+      client_secret:
+        type: string
+        order: 2
+        title: Client secret
+        airbyte_secret: true
+    additionalProperties: true
+
+metadata:
+  autoImportSchema:
+    List Users: true
+    Variants: true
+    List Products: true
+    Payments: true
+    Payment Methods: true
+    List Orders: true
+    Menus: true
+    Addresses: true
+    Classifications: true
+    Adjustments: true
+    CMS Pages: true
+    CMS Sections: true
+    Shipments: true
+    Shipping Categories: true
+    States: true
+    Stock Items: true
+    Stock Locations: true
+    Stock Credit Categories: true
+    Store Credit Types: true
+    Store Credits: true
+    Tax Categories: true
+    Tax Rates: true
+    Wished Items: true
+    Wishlists: true
+    Zones: true
+    Webhook Subscribers: true
+    Taxonomies: true
+    Taxons: true
+  testedStreams:
+    List Users:
+      streamHash: 81c15fe0f62280c1681ea8b538bacef3f52b115c
```

Comment:
All streams weren't tested. I strongly suggest to build the connector using Airbyte Cloud.